client-app/modules/customer-reviews/components/product-reviews.vue

```diff
@@ -45,6 +45,22 @@
           <div>
             {{ review.review }}
           </div>
+
+          <div v-if="review.images?.length" class="image-gallery__thumbs-container">
+            <Swiper
+              :slides-per-view="MAX_REVIEW_IMAGES_COUNT"
```

Comment:
what if hey 10? You want to show all of them in a row?

Reply:
We have 5 images limit on the backend right now. BTW I'll add a Prev/Next navigation. 5 images are too much for mobile version.